index.js

```diff
@@ -42,12 +42,13 @@ co.wrap = function (fn) {
 
 function co(gen) {
   var ctx = this;
+  var args = slice.call(arguments, 1)
```

Comment:
What about inlining slicing using `new Array(arguments.length); (...)` in order to avoid v8 deoptimization?